projects/portal/src/app/views/staking/edit-validator-simple/edit-validator-simple.component.html

```diff
@@ -0,0 +1,11 @@
+<h2>Upload wallet-backup file here to edit Validator information</h2>
+<form class="h-screen" #formRef="ngForm">
+  <input
+    #fileInputRef
+    name="fileInput"
+    type="file"
+    multiple="false"
+    accept="image/*"
+    (change)="onChangeFile($event)"
+  />
+</form>
```

Comment:
You can edit the design of file input button by two steps:

- Set the visibility of file input button as `hidden`
- Create simple normal button with onClick handler. If it is clicked, JS code for emitting click event of **file input button** will be evoked.

Reply:
Thank you for your comment.
Done as below.
![image](https://user-images.githubusercontent.com/75844498/172042994-2d439da9-2cd8-4d2a-9257-9a21e8e8e681.png)